pkg/volume/fc/attacher.go

```diff
@@ -95,39 +96,42 @@ func (attacher *fcAttacher) GetDeviceMountPath(
 	return attacher.manager.MakeGlobalPDName(*mounter.fcDisk), nil
 }
 
-func (attacher *fcAttacher) MountDevice(spec *volume.Spec, devicePath string, deviceMountPath string) error {
-	mounter := attacher.host.GetMounter(fcPluginName)
-	notMnt, err := mounter.IsLikelyNotMountPoint(deviceMountPath)
-	if err != nil {
-		if os.IsNotExist(err) {
-			if err := os.MkdirAll(deviceMountPath, 0750); err != nil {
+func (attacher *fcAttacher) MountDevice(spec *volume.Spec, devicePath string, deviceMountPath string) (volumetypes.OperationStatus, error) {
+	mountInternal := func() error {
+		mounter := attacher.host.GetMounter(fcPluginName)
+		notMnt, err := mounter.IsLikelyNotMountPoint(deviceMountPath)
+		if err != nil {
+			if os.IsNotExist(err) {
+				if err := os.MkdirAll(deviceMountPath, 0750); err != nil {
+					return err
```

Comment:
it does not need to return OperationStatus?

Reply:
It does return `OperationStatus` by calling `return  volumetypes.OperationFinished, mountInternal()` at the end. Some of this refactoring was hard to do in individual plugins because changing  return value of a volume interface function means modifying all those return statements in different places.  I have wrapped original `mountDevice` in internal `mountInternal` function and then we are returning both `OperationStatus` and `Error.